docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+web:
+  build: .
+  ports:
+    - "8888:8888"
+  volumes:
+    - .:/src
```

Comment:
right, this is just for development currently - we will have to seperate out dev and production configs eventually. But for the purposes of getting ramped up I think its fine for now. I'll create an issue to set up production configs